woocommerce-gateway-ebanx/assets/css/paying-via-ebanx.css

```diff
@@ -296,10 +296,7 @@ li[class*="ebanx"] .payment_box .ebanx-credit-card-label > .ebanx-credit-card-bi
 .woocommerce .ebanx-button {
   max-width: 100%;
   display: inline-block;
-  padding-top: .75rem;
-  padding-right: 1rem;
-  padding-bottom: .75rem;
-  padding-left: 1rem;
+  padding: .75rem 1rem;
```

Comment:
@gpressutto5 be careful here... Sometimes I needed add the padding in this way to avoid conflicts, so, *always* test it!